src/core/lwip/tcp_out.c

```diff
@@ -234,6 +235,33 @@ static struct tcp_seg *tcp_create_segment(struct tcp_pcb *pcb, struct pbuf *p, u
     return seg;
 }
 
+/**
+ * Allocate a PBUF_RAM pbuf, perhaps with oversize space at the end.
+ *
+ * This function is like pbuf_alloc(layer, length, PBUF_RAM) except
+ * there may be extra bytes available at the end.
+ *
+ * @param length size of the pbuf's payload.
+ * @param oversize pointer to a u16_t that will receive the number of usable tail bytes.
+ * @param pcb The TCP connection that willo enqueue the pbuf.
+ * @param
+ */
+static struct pbuf *tcp_pbuf_prealloc_express(u16_t length, u16_t *oversize, struct tcp_pcb *pcb,
```

Comment:
oversize is not used in ZC flow... i think you can remove it from all express api

Reply:
I'll look at it.